### PR TITLE
Fix bus reconstruct assert

### PR DESCRIPTION
**Pull request by alaindargelas**
_What are the reasons/motivation for this change?_

_Explain how this is achieved._

_If applicable, please suggest to reviewers how they can test the change._

### CODE REVIEW

File: passes/cmds/reconstructbusses.cc [hunk 1]
```diff
@@ -234,10 +296,15 @@ struct ReconstructBusses : public ScriptPass {
...
 				run_pass("write_rtlil post_reconnect_top.rtlil");
-			// Remove old wires
+			// Remove old bit blasted wires
 			// Cleans the dangling connections too
-			log("Removing old wires\n");
+			log("Removing bit blasted wires\n");
 			log_flush();
+			if (debug) {
+				for (RTLIL::Wire* w : wires_to_remove) {
+				  std::cout << "  " << w->name.c_str() << std::endl;
```
Comment (akashlevy) [inline passes/cmds/reconstructbusses.cc:305]: Fix whitespace on this line and next

**Review by akashlevy — CHANGES_REQUESTED** [review]
Have you tested this on a full run of `pqlib` with (1) Yosys port bit-blasting, and (2) internal net renaming?

**Comment by alaindargelas** [comment]
@akashlevy still running the validation, see Slack

**Comment by alaindargelas** [comment]
@akashlevy , this is validated

**Review by akashlevy — APPROVED** [review]

**Pull request merged**